docs/Server/API/V1/relay.md

```diff
@@ -0,0 +1,143 @@
+# `POST /api/v1/relay/<uri>`
+
+This API creates a dataset resource by reading data from a Relay server in two
+steps. The provided URI represents a "manifest file" stored on the Relay server,
+a JSON file (`application/json` MIME format) defining the original tarball name,
+the MD5 hash value, optional metadata key/value pairs for the dataset, and a
+URI from which the Pbench Server expects to `GET` an `application/octet-stream`
+of a file created by `tar` and compressed with the `xz` program.
+
+Primarily this is expected to be a native Pbench Agent tarball with a specific
+structure; however with the `server.archiveonly` metadata key the Pbench Server
+can be used to archive and manage metadata for any tarball passed through a
+Relay server.
+
+## URI parameters
+
+`<uri>` string \
+The Relay server URI of the tarball's manifest `application/json` file. This
+must provide the following JSON keys:
+
+For example,
+
+```json
```

Comment:
The key list is missing.  (Perhaps we should combine the "For example" into the end of the previous sentence?)

Or, should the example follow the `## Manifest file keys` section?

Reply:
I put the example first as a reference for the detailed list, but the original "the following JSON keys" got left behind. Sigh.